site/src/component/Review/ReviewCard.scss

```
@@ -211,7 +211,7 @@
     font-size: 20px;
   }
   .colored-upvote {
-    color: var(--blue-primary);
+    color: var(--mui-palette-primary-main);
   }
   .colored-downvote {
     color: var(--red-primary);
```

Don't forget to update this `--red-primary`, which was recently added in #882 

```suggestion
    color: var(--mui-palette-error-main);;
```